### ISSUE TITLE
Greenkeeper not working

### ISSUE THREAD

**Issue by YerkoPalma** [choojs/choo-umd#4]
Ok, so Grrenkeeper isn't opening any PR. I activated the github bot, not sure if there is anything else to do, not a big user of greenkeeper here haha.
So any help on activating the service will be very welcome @choojs/trainspotters

**Comment by goto-bus-stop** [comment]
Greenkeeper only opens PRs for new versions that break tests, or that don't satisfy the constraints in package.json (so major versions basically). Otherwise it creates a branch, waits for Travis, and delete the branch on success. If its not opening prs, its probably because there were no breaking updates

**Comment by yoshuawuyts** [comment]
Sounds like we should create a bot of sorts for this, perhaps.

**Comment by yoshuawuyts** [comment]
@YerkoPalma any idea on what we should do?

**Comment by bcomnes** [comment]
I think removing the ^ range might fix it.

**Comment by yoshuawuyts** [comment]
@bcomnes interesting — that would force us to publish lots of patch
releases, but wouldn't influence people upstream. Not half bad.y

**Comment by YerkoPalma** [comment]
Patch releases in this module right? not in the main choo module? if that's the case, it is ok to me

**Comment by bcomnes** [comment]
That or set up pre-publish scripts in choo proper.  I know I leaned to a separate repo, but now I'm thinking that might be the cleanest approach than rely on GK which can sometimes be unreliable.

**Comment by yoshuawuyts** [comment]
Any ideas on what to do next?

**Comment by YerkoPalma** [comment]
@bcomnes I agree that a pre-publish hook on main choo repo would be a better approach, but I'm not sure if that would affect people doing `npm install`. Because if someone is using browserify or other bundler I don't think they want to load the umd build too.

**Comment by goto-bus-stop** [comment]
I also think that it's best to move this into the main choo module. outputting a umd bundle into its `dist/` folder on `prepublishOnly`.
